modules/core/src/main/scala/com/azavea/stac4s/extensions/eo/Band.scala

```diff
@@ -0,0 +1,35 @@
+package com.azavea.stac4s.extensions.eo
+
+import cats.Eq
+import eu.timepit.refined.types.string.NonEmptyString
+import io.circe._
+import io.circe.refined._
+
+case class Band(
+    name: NonEmptyString,
+    commonName: NonEmptyString,
+    description: NonEmptyString,
```

Comment:
As you noted, the spec says these are all optional and I agree that seems crazy.

I would vote for only requiring `name` though because it would be nice to be able to use the extension when we know we have multiband imagery but not an easy or straightforward way of getting these other attributes. In `stac-viewer` we can use the name or common name for display purposes and it would still be useful.

Reply:
0feb030